opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/aggregation/AggregationQueryBuilderTest.java

```diff
@@ -28,7 +28,9 @@
 import static org.opensearch.sql.opensearch.utils.Utils.group;
 import static org.opensearch.sql.opensearch.utils.Utils.sort;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.gson.JsonParser; 
```

Comment:
```suggestion
import com.google.gson.JsonParser;
```

Reply:
You have similar mistake in few more places.
Run `git diff --check upstream/main`